### PR TITLE
highlighted chat user visibility, fixes #1026

### LINKED ISSUE #1026: [Bug]: Poor visibility of username who wrote the last message shown in chat preview on left side panel when highlighted
### What Monal Release channel are you using?

Alpha

### iOS system version

_No response_

### iOS Monal version

_No response_

### macOS system version

14.4.1

### macOS Monal version

Alpha 51b894a1b0 (Mar 28 2024 08:48:16 UTC)

### Used XMPP server (domain)

disroot.org

### Which XMPP-Server software are you using?

Prosody

### XMPP Server Software Version

_No response_

### How many accounts are you using in Monal?

3

### What happened?

In the left side chat preview panel users can see:
- channel
- username posting last message
- last message

With https://github.com/monal-im/Monal/issues/756 resolved and the selected chat highlighted, the username is not visible.

When highlighted the color for username should be changed, so that it still has good contrast and can be read.

Affects 1:1 chats as well as MUCs.

### Anything else?

_No response_

### FAQ

- [X] I have [checked the FAQ](https://github.com/monal-im/Monal/wiki/FAQ---Frequently-Asked-Questions) for my issue

### Considerations for XMPP users

- [X] I have checked if my issue can be solved with [Considerations for XMPP users](https://github.com/monal-im/Monal/wiki/Considerations-for-XMPP-users) and [Considerations for XMPP server admins](https://github.com/monal-im/Monal/wiki/Considerations-for-XMPP-server-admins)

### Related Issues

- [X] I have cross-checked this overview https://github.com/monal-im/Monal/issues/322 as well as filtered for related labels https://github.com/monal-im/Monal/labels

### XEP-Check

- [X] I have checked that at least XEP-198, XEP-0280, XEP-0352, XEP-0357, XEP-0313 and XEP-0163 are activated on my server and shown as 'green' under Settings --> Account--> (i) in advanced settings

### Notifications-Menu

- [X] I have checked that all checkmarks are present under Settings --> Notifications

### PR DESCRIPTION
**Pull request by zainashraf1562**
issue #1026 
Video Clip:
https://github.com/monal-im/Monal/assets/155921778/58234894-04a3-4bee-97af-067c13956181

### CODE REVIEW

**Comment by zainashraf1562** [comment]
Also another bug i just noticed to show users last message for the 'Me' we don't include ':' to separate 'Me' from last message.
Screenshot 2024-04-02 at 3 39 06 PM

File: Monal/Classes/ActiveChatsViewController.m [hunk 1]
```diff
@@ -753,10 +753,14 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     cell.selectionStyle = UITableViewCellSelectionStyleNone;
     
     // Highlight the selected chat
-    if([MLNotificationManager sharedInstance].currentContact != nil && [chatContact isEqual:[MLNotificationManager sharedInstance].currentContact]) 
+    if([MLNotificationManager sharedInstance].currentContact != nil && [chatContact isEqual:[MLNotificationManager sharedInstance].currentContact]){
         cell.backgroundColor = [UIColor lightGrayColor];
-    else
+        cell.statusText.textColor = [UIColor whiteColor];
+    }
+    else{
```
Comment (tmolitor-stud-tu) [inline Monal/Classes/ActiveChatsViewController.m]:
```suggestion
    else
    {
```

File: Monal/Classes/ActiveChatsViewController.m [hunk 2]
```diff
@@ -753,10 +753,14 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     cell.selectionStyle = UITableViewCellSelectionStyleNone;
     
     // Highlight the selected chat
-    if([MLNotificationManager sharedInstance].currentContact != nil && [chatContact isEqual:[MLNotificationManager sharedInstance].currentContact]) 
+    if([MLNotificationManager sharedInstance].currentContact != nil && [chatContact isEqual:[MLNotificationManager sharedInstance].currentContact]){
```
Comment (tmolitor-stud-tu) [inline Monal/Classes/ActiveChatsViewController.m]:
```suggestion
    if([MLNotificationManager sharedInstance].currentContact != nil && [chatContact isEqual:[MLNotificationManager sharedInstance].currentContact])
```

File: Monal/Classes/ActiveChatsViewController.m [hunk 3]
```diff
@@ -753,10 +753,14 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
...
         cell.backgroundColor = [UIColor lightGrayColor];
-    else
+        cell.statusText.textColor = [UIColor whiteColor];
+    }
```
Comment (tmolitor-stud-tu) [inline Monal/Classes/ActiveChatsViewController.m]:
```suggestion
```

**Review by tmolitor-stud-tu — CHANGES_REQUESTED** [review]
Please apply my suggestions to fix the style according to our coding style guide in our wiki.

**Comment by tmolitor-stud-tu** [comment]
I've changed the style myself and merged it, see here: https://github.com/monal-im/Monal/commit/98a5aac201f095917d48ebaac211c811da0ff5ea

**Comment by zainashraf1562** [comment]
> Please apply my suggestions to fix the style according to our coding style guide in our wiki.

Ah Apologies creature of habit 😭 will be more mindful in the future.

**Pull request merged**